### DISCUSSION TITLE
Spaces in paths

### DISCUSSION THREAD

**Discussion by kenNash** [johnlindquist/kit#1403] [Q&A]
Hi Kiters!

In Terminal.app, 

```
open ~/Desktop/A\ B
```

correctly opens my "A B" folder. But

```
exec(`open ~/Desktop/A\ B`)
```

in a script doesn't.

Am I doing something wrong?
Is it a bug?

**Comment by johnlindquist** [comment]
@kenNash Here's how I would write it:

```js
let folderPath = home("Desktop", `"A B"`)
await exec(`open ${folderPath}`)
```

Using the `home` helper (or `path.resolve`, etc) as a cross-platform/shell-safe replacement for `~`. You also separate the strings in an array rather than a using a system specific path separator. And this also allows you to use quoted paths like above.

**Reply by kenNash** [reply]
Thank you @johnlindquist!
